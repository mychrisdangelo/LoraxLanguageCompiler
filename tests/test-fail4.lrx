/* Test fucntion return types */
int function()
{
	return true;
}
int main () {
	function();
}