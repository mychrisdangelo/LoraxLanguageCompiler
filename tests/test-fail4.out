Fatal error: exception Failure("function return type bool; type intexpected")
